.gitignore

```diff
@@ -47,3 +47,4 @@ install/kubernetes/istio-sidecar-injector.yaml
 install/kubernetes/istio.yaml
 samples/bookinfo/consul/bookinfo.sidecars.yaml
 samples/bookinfo/eureka/bookinfo.sidecars.yaml
+istio.VERSION
```

Comment:
I realize I mentioned to add it to git ignore but I am hoping it still is editable despite that - the bot still updates the proxy sha in there I think